### PR TITLE
Mybuild

### PR DESCRIPTION
**Pull request by mt819**

### CODE REVIEW

**Comment by aplgithub** [comment]
Hello @mt819 ; Thank you this is really great.  There are a few more things if you have time...

* ```aboutcontent.html```, ```aboutcontent_de.html```, ... -- add your name to the list of people doing translations.
* Files in ```../haikudepotserver-webapp/src/main/webapp/js/app/passivecontent``` require localization.
* Files in ```../haikudepotserver-core/src/main/resources/mail``` require localization.
* ```unsupported.jsp``` has localizations
* ```ErrorServlet.java``` has localizations

**Comment by mt819** [comment]
> Hello @mt819 ; Thank you this is really great. There are a few more things if you have time...
> 
>     * `aboutcontent.html`, `aboutcontent_de.html`, ... -- add your name to the list of people doing translations.
> 
>     * Files in `../haikudepotserver-webapp/src/main/webapp/js/app/passivecontent` require localization.
> 
>     * Files in `../haikudepotserver-core/src/main/resources/mail` require localization.
> 
>     * `unsupported.jsp` has localizations
> 
>     * `ErrorServlet.java` has localizations
Thanks for suggestion, I will add files to translate.

**Comment by aplgithub** [comment]
Thanks; I notice that some of the files such as...

* ```viewrepositorysource_ja.html```
* ```pkgscreenshotarchiveimporthelp_ja.html```

...are not translated.  Can you please remove those if you are not going to translate them because the system will fall back to the English language resources anyway.

**Comment by mt819** [comment]
Hi, I finished the first stage of translation. I will update translations later.

File: haikudepotserver-webapp/src/main/webapp/js/app/passivecontent/pkgscreenshotarchiveimporthelp_ja.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!--
+  ~ Copyright 2017, Andrew Lindesay
```
Comment (aplgithub) [inline haikudepotserver-webapp/src/main/webapp/js/app/passivecontent/pkgscreenshotarchiveimporthelp_ja.html]: Can you please replace this copyright with your own name.

Comment (mt819) [reply haikudepotserver-webapp/src/main/webapp/js/app/passivecontent/pkgscreenshotarchiveimporthelp_ja.html]: Hi, I updated copyright.

**Comment by aplgithub** [comment]
Hello; Thank you again.  Your changes have been merged and will go into the next release.

**Pull request merged**